test/integration/testManualConnection.js

```diff
@@ -265,7 +266,7 @@ if (process.env.RUN_MANUAL_TESTS_ONLY === 'true') {
     });
   });
 
-  describe.only('keepAlive test', function () {
+  describe('keepAlive test', function () {
```

Comment:
should we skip only here?

Reply:
yes. in manual tests it depends on what we currently testing ....